content/docs/command-reference/unprotect.md

```diff
@@ -84,6 +84,6 @@ $ ls -lh
 -rw-r--r--  1  120B Nov 27 13:29 Posts.xml.zip.dvc
 -rw-r--r--  1   10M Nov 27 13:30 Posts.xml.zip
 
-$ ls -lh ls -lh .dvc/cache/ce/
+$ ls -lh ls -lh .dvc/cache/files/md5/ce/
```

Comment:
Not related to this PR but looks like there was a issue while Copy-pasting here.
```suggestion
$ ls -lh .dvc/cache/files/md5/ce/
```